lib/core/versions/latest/DocumentComposer.ts

```diff
@@ -124,35 +124,35 @@ export default class DocumentComposer {
    * Validates the schema of the given full document.
    * @throws SidetreeError if given document patch fails validation.
    */
-  public static validateDocument (document: any) {
+  private static validateDocument (document: any) {
```

Comment:
I turned this into private because I don't see it used anywhere lmk if this is not ok

Reply:
Looks good here.